Snippets/ASB/ASB_7/Sanitization.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using NServiceBus;
+using NServiceBus.AzureServiceBus;
+using NServiceBus.AzureServiceBus.Addressing;
+
+[SuppressMessage("ReSharper", "UnusedMember.Local")]
+public class Sanitization
+{
+    void EndpointOrientedTopologySanitization(EndpointConfiguration endpointConfiguration)
+    {
+        #region asb-endpointorientedtopology-sanitization
+
+        var transport = endpointConfiguration.UseTransport<AzureServiceBusTransport>();
+
+        transport.Sanitization().UseStrategy<EndpointOrientedTopologySanitization>();
+
+        #endregion
+    }
+
+
+    void CustomSanitization(EndpointConfiguration endpointConfiguration)
+    {
+        #region asb-custom-sanitization
+
+        var transport = endpointConfiguration.UseTransport<AzureServiceBusTransport>();
+
+        transport.Sanitization().UseStrategy<CustomSanitization>();
```

Comment:
Not this time around. The transport using strategies for other things as well. Having a single configuration not consistent with the rest will feel odd, if not off.